index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0
```

Comment:
technically, for THIS case (an empty array) this works, but what we want is for the length to be as adaptable as it can; this can be achieved by setting it as `this.items.length` which will read the length of your array whether it is empty or not.

Reply:
Where you needed length was in your constructor and in the add() to update it (basically putting the same thing twice but it updates it since we add another value to your array); also in certain conditionals for some code to run but that was just referencing the length property of your constructor. It didn't work in the constructor nor the function?